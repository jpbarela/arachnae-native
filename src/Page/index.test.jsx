// @flow
import * as React from "react";
import { render } from "@testing-library/react-native";
import { Page } from "./index";

describe("Page", () => {
  it("renders", () => {
    const container = render(<Page />);
    expect(container).toMatchSnapshot();
  });
});
