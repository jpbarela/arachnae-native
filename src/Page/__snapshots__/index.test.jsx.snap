// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page renders 1`] = `
<View
  style={
    Array [
      Object {
        "height": "100%",
        "width": "100%",
      },
      Object {
        "backgroundColor": "white",
      },
    ]
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "flexDirection": "column",
        "justifyContent": "center",
      }
    }
  />
</View>
`;
